src/core/model_repository_manager.cc

```diff
@@ -871,22 +871,20 @@ ModelRepositoryManager::Create(
         "Unexpected initial state for model repository");
   }
 
+  Status status = Status::Success;
   for (const auto& name : added) {
-    ModelConfig model_config;
-    RETURN_IF_ERROR(local_manager->GetModelConfig(name, &model_config));
-    RETURN_IF_ERROR(
-        local_manager->status_manager_->InitForModel(name, model_config));
-
-    std::vector<int64_t> versions;
-    RETURN_IF_ERROR(
-        local_manager->VersionsToLoad(name, model_config, versions));
-    RETURN_IF_ERROR(local_manager->backend_life_cycle_->AsyncLoad(
-        name, versions, model_config));
+    // If there is error on model loading, just report it and move to next model
+    Status update_status = local_manager->Update(name, true);
```

Comment:
Is this something that we should have caught in testing? Can you add a lifecycle test to make sure it doesn't regress?